zipline/testing/core.py

```diff
@@ -49,7 +49,9 @@
 from zipline.utils import security_list
 from zipline.utils.calendars import get_calendar
 from zipline.utils.input_validation import expect_dimensions
-from zipline.utils.numpy_utils import as_column
+from zipline.utils.numpy_utils import (
```

Comment:
`fixtures.py` I'll change back - there were leftover changes to the imports from when I thought I'd be moving things there. I don't see how we can get away with not editing `core.py`, however; there are lots of changes further down in the file where we refactor certain logic to reuse it in other places.